content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -98,7 +98,8 @@ each source also represents a single path within that volume. The three sources
 
 1. A `serviceAccountToken` source, that contains a token that the kubelet acquires from kube-apiserver.
    The kubelet fetches time-bound tokens using the TokenRequest API. A token served for a TokenRequest expires
-   either when the pod is deleted or after a defined lifespan (by default, that is 1 hour).
+   either when the pod is deleted or after a defined lifespan (by default, that is 1 hour). The token
+   will be refreshed by the kubelet prior to expiry.
```

Comment:
We can avoid the need to pick:
```suggestion
   either when the pod is deleted or after a defined lifespan (by default, that is 1 hour).
   The kubelet also refreshes that token before the token expires.
```

(we prefer to [use the active voice](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice))